integration/base/integration_test.go

```diff
@@ -87,16 +87,16 @@ var _ = Describe("ship app", func() {
 					buf := new(bytes.Buffer)
 					cmd.SetOutput(buf)
 					args := []string{
-						"app",
-						"--headless",
-						fmt.Sprintf("--runbook=%s", path.Join(testInputPath, ".ship/release.yml")),
+						"init", "staging.replicated.app/integration",
+						fmt.Sprintf("--runbook=%s", path.Join(testInputPath, ".ship/ship.yml")),
 						fmt.Sprintf("--customer-id=%s", testMetadata.CustomerID),
 						fmt.Sprintf("--installation-id=%s", testMetadata.InstallationID),
```

Comment:
Couple things
### online tests
 I think  for online tests, `customer-id` and `installation-id` should be expected to be in  the querystring, not passed as flags. e.g. `"staging.replicated.app/integration?customer_id=xxxx&installation_id=xxxxx"`

I don't think we should support these `--customer-id` and `--installation-id` flags for `ship init`, but leaving them in is fine for now. 

### Offline tests

For offline tests, we shouldn't use the `--runbook` flag, we should `ship init` on the runbook file `path.Join(testInputPath, ".ship/ship.yml")`

Reply:
I guess if --customer-id is for local testing of template functions like `{{repl Installation "customer_id"}}` then it should stay, although potentially with a different flag name. the other use `--set-xxx` which seems to imply more that we're faking something for local dev. 

need to think about this one some more